### PR TITLE
cartridge-cli 2.0.0

### PR DESCRIPTION
**Pull request by dokshina**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/cartridge-cli.rb [hunk 1]
```diff
@@ -12,13 +12,28 @@ class CartridgeCli < Formula
...
+      ldflags = [
+        "-s", "-w",
+        "-X #{package}/version.gitTag=#{tag}",
+        "-X #{package}/version.gitCommit=#{commit}"
+      ]
+
+      system "go", "mod", "download"
+      system "go", "build", "-o", bin/"cartridge", "-ldflags", ldflags.join(" "),
+          "github.com/tarantool/cartridge-cli/cli"
+    end
```
Comment (bayandin) [inline Formula/cartridge-cli.rb:36]: We're working on `GOPATH` usage deprecation in homebrew-core (https://github.com/Homebrew/homebrew-core/issues/47627), so I believe we could simplify this installation like

```suggestion
    commit = Utils.safe_popen_read("git rev-parse --short HEAD").chomp

    ldflags = %W[
      -s -w
      -X github.com/tarantool/cartridge-cli/cli/version.gitTag=#{version}
      -X github.com/tarantool/cartridge-cli/cli/version.gitCommit=#{commit}
    ]

    system "go", "build", "-o", bin/"cartridge", "-ldflags", ldflags.join(" "), "cli/main.go"
```

**Comment by dokshina** [comment]
Thanks a lot for your suggestion! 
I've run `brew test` and `brew audit` locally for updated formula, everything is OK.

**Comment by bayandin** [comment]
Could you please also squash all the commits into single `cartridge-cli 2.0.0`

**Comment by dokshina** [comment]
Yes, done.

**Review by bayandin — DISMISSED** [review]
Thanks @dokshina for your contribution to Homebrew! 🎉 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Review by dawidd6 — DISMISSED** [review]

**Comment by dawidd6** [comment]
@bayandin sorry for dismissing your approval, had to test the Action :)

**Pull request closed without merging**